src/utils/jellyfin-apiclient/getItems.ts

```diff
@@ -0,0 +1,74 @@
+import type { BaseItemDtoQueryResult } from '@jellyfin/sdk/lib/generated-client';
+import { ApiClient } from 'jellyfin-apiclient';
+
+interface GetItemsRequest {
+    Ids?: string;
+    Limit?: number;
+}
+
+const ITEMS_PER_REQUEST_LIMIT = 25;
```

Comment:
Based on a maximum url length around 2000 characters, I think we can pretty safely bump this up to `40`. In my testing that put the url length around 1500 characters which should leave plenty of headroom for variable domain / base urls and differing parameter sets.

Reply:
Good point. I'm not sure what the smallest default url length of the widely used webservers is. According to a quick google search it should be at least 4kb, so we might be able to bump it even higher, but I don't think the performance issues caused by too many requests should be too big even with "only" 40. I have the original version of this PR in use for a while now with a season of around 1000 episodes and it was never an issue (Syncplay).

I bumped it to 40 now.